### PR TITLE
fixes #98. Modification of the max walk distance and …

### LINKED ISSUE #98: USF PATS request - Change default max walk in trip planner GUI
After talking with USF Parking and Transportation Services (PATS) today, we should change the default "Maximum walk" parameter in the trip planning screen from `1.00` to `0.3`.  At values from `1.0` to `0.4`, OTP only returns a single trip itinerary (walk).  When you change to `0.3`, it provides at least 2 trip itineraries, where the first is walk and the 2nd is Bull Runner.  USF PATS walks to always display the bus option along with walk if possible.

@jailby Could you please tackle this?

### PR DESCRIPTION
**Pull request by jailby**
cleanup of old attributes not present in upstream OTP anymore.

### CODE REVIEW

File: src/client/js/otp/modules/planner/PlannerModule.js [hunk 1]
```diff
@@ -107,12 +105,6 @@ otp.modules.planner.PlannerModule =
         
         this.defaultQueryParams = _.clone(otp.modules.planner.defaultQueryParams);
 
-        if (otp.config.metric) {
```
Comment (barbeau) [inline src/client/js/otp/modules/planner/PlannerModule.js:110]: Per https://github.com/CUTR-at-USF/usf-mobullity/issues/98#issuecomment-149992750, I think this block should stay - it still exists upstream.

File: src/client/js/otp/modules/planner/PlannerModule.js [hunk 2]
```diff
@@ -21,9 +21,7 @@ otp.modules.planner.defaultQueryParams = {
     date                            : moment().format(otp.config.locale.time.date_format),
     arriveBy                        : false,
     mode                            : "TRANSIT,WALK",
-    maxWalkDistance                 : 1609.34, // 1 mile
```
Comment (barbeau) [inline src/client/js/otp/modules/planner/PlannerModule.js:24]: Per my comment https://github.com/CUTR-at-USF/usf-mobullity/issues/98#issuecomment-149992750, I think these two variables should stay - they still exist upstream.

File: src/client/js/otp/modules/planner/PlannerModule.js [hunk 3]
```diff
@@ -21,9 +21,7 @@ otp.modules.planner.defaultQueryParams = {
...
-    metricDefaultMaxWalkDistance    : 750, // meters
-    imperialDefaultMaxWalkDistance  : 1609.34, // 1 mile
+    maxWalkDistance                 : 482.8032, // .3 mile unit are in meter
```
Comment (barbeau) [inline src/client/js/otp/modules/planner/PlannerModule.js:25]: This comment for the unit change looks good.

**Comment by barbeau** [comment]
@jailby Per https://github.com/CUTR-at-USF/usf-mobullity/issues/98#issuecomment-149992750, let's leave the other items, and just change the value for `maxWalkDistance` and the comment for that variable.

You should be able to make these changes in this PR by re-staging these changes in git, and then only modifying only the one line, and committing this.  You can do this by checking out the same branch, and then:

`git reset HEAD^`

...then do the modifications so only the one line is changed, then commit it again.  Then do:

`git push -f origin maxWalk`

...and it should replace this commit with the new commit that only changes the one line.

**Comment by jailby** [comment]
Just changing `maxWalkDistance` will not fix #98.

Since it goes through the condition at L110, `maxWalkDistance` will be updated with either to the value of `metricDefaultMaxWalkDistance` or `imperialDefaultMaxWalkDistance`.

I have to also update `imperialDefaultMaxWalkDistance` to 482.8032 meters to make sure that it correspond to 0.3 miles.

**Comment by jailby** [comment]
@barbeau I made another commit to link the question I asked about the three attributes. I think this branch should be merged now. Let me know what you think.

**Comment by barbeau** [comment]
Thanks @jailby!

**Pull request merged**